test/functional/aws_standalone_vpc.go

```diff
@@ -160,18 +170,49 @@ func TestStandaloneVPCExists(t *testing.T, testingCtx *common.TestingContext) {
 		t.Skip("_network key does not exist in aws strategy configmap, skipping standalone vpc network test")
 	}
 
-	// get the vpc cidr block
-	expectedCidr, err := getCidrBlock(strat)
-	if err != nil {
-		t.Fatal("could not get cidr block", err)
-	}
-
 	// get the cluster id used for tagging aws resources
 	clusterTag, err := getClusterID(ctx, testingCtx.Client)
 	if err != nil {
 		t.Fatal("could not get cluster id", err)
 	}
 
+	// get the vpc cidr block
+	expectedCidr, err := getCidrBlockFromStrategyMap(strat)
+	if err != nil {
+		t.Fatal("could not get cidr block from strategy map", err)
+	}
+
+	vpcCidrBlock := expectedCidr
+	// if the cidr strategy map is empty then attempt to retrieve the cidr block from the vpc
+	if expectedCidr == "" {
+		vpcCidrBlock, err = getVpcCidrBlock(ec2Sess, vpcClusterTagKey, clusterTag)
```

Comment:
```suggestion
		standaloneCidr, err = getVpcCidrBlock(ec2Sess, vpcClusterTagKey, clusterTag)
```

matching `clusterCidr` so that it's clear how the two differ if reading later.